test/kb_orthofinder_genome_test.tmp

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
Was this checked in erroneously?

Reply:
Maybe this should be added to a different directory and labeled different so it is not run with kb-sdk test. Then a description of the what the script is for could be added to the top of the file.

Reply:
Well, it doesn't run with kb-sdk test unless you rename it with the *.py suffix. Nevertheless, I saw how GFU gives you the option of choosing different test scripts to run, so I may look into that at some point. Nobody needs to run this, but it's for me to test a user's genome.

Reply:
The tests folder should be for files that are tests, artificats, config files, or documentation, so placing it here would be breaking that flow.